repositories/events.js

```diff
@@ -0,0 +1,92 @@
+const {Op} = require('sequelize');
+const { Event, User } = require('../models');
+
+const EventsRepository = {
+  find: async (location) => {
+    try {
+      let events = [];
+      if (!location || location === 'all') {
+        events = await Event.findAll({
```

Comment:
here is better to use `if` to create query params and so have one call to db (in terms of code). This would reduce code duplication and would allow us to get rid of using `let`. Try always avoid using `let`